downstream/modules/platform/ref-controller-AD-and-kerberos-credentials.adoc

```diff
@@ -0,0 +1,11 @@
+[id="ref-controller-AD-and-kerberos-credentials"]
+
+= AD and Kerberos Credentials
+
+Active Directory only:
+
+* If you are only planning to run playbooks against Windows machines with AD usernames and passwords as machine credentials, you can use "user@<domain>" format for the username and an associated password.
```

Comment:
```suggestion
* If you are only planning to run playbooks against Windows machines with AD usernames and passwords as machine credentials, you can use the "user@<domain>" format for the username and an associated password.
```
Maybe delete and an associated password", because it might look like it too can use the format.